src/components/navigationLoader/NavigationLoaderExample.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```

Comment:
Why the name `NavigationLoader`? It does not load anything. I would rename to something like `NavigationLoadingExample` or `LoadingNavigationExample`.